src/core/doe_commercial_reference_building_loads.jl

```diff
@@ -296,3 +296,51 @@ function custom_normalized_flatload(doe_reference_name, year)
     return normalized_profile
 end
 
+"""
+    get_monthly_energy(; power_profile::AbstractArray{<:Real,1}=Real[],
+                        normalized_profile::AbstractArray{<:Real,1}=Real[],
+                        annual_energy::Float64=nothing,
+                        year::Int64=2017)
+Get monthly energy from an hourly load profile.
+"""
+function get_monthly_energy(; power_profile::AbstractArray{<:Real,1}=Real[],
+                            normalized_profile::AbstractArray{<:Real,1}=Real[],
+                            annual_energy::Float64=nothing,
+                            year::Int64=2017)
```

Comment:
it looks like `simulated_load.jl` only uses the argument `power_profile` and the other arguments aren't used anywhere else.  Do we need the rest of these?  If the answer is no, disregard the other comments in this function and I can try to simplify.

Reply:
I'm taking a stab at simplifying. Let me know if it looks like could be simplified more.